### PR TITLE
Allow user to set OPENJCEPLUS_GIT_REPO and BRANCH

### PR DESCRIPTION
**Pull request by llxia**
resolves: https://github.com/adoptium/aqa-tests/issues/5055

### CODE REVIEW

**Comment by karianna** [comment]
@llxia This fails because the newer upload artifact action doesn't allow the uploading of the exact same named artifact.

**Comment by llxia** [comment]
PR builds failure has nothing to do with this PR. It is a known issue that PR builds do not check up PR repo: https://github.com/adoptium/aqa-tests/issues/4855. Even if it does, this PR cannot be tested via PR build.

File: functional/OpenJcePlusTests/build.xml [hunk 1]
```diff
@@ -28,6 +28,7 @@
 	<property name="build" location="./bin" />
 	<property name="LIB" value="junit4,hamcrest_core,bcprov_jdk18on"/>
 	<import file="${TEST_ROOT}/TKG/scripts/getDependencies.xml"/>
+	<property environment="env" />
```
Comment (JasonFengJ9) [inline functional/OpenJcePlusTests/build.xml:31]: This branch is [1 commit ahead of](https://github.com/llxia/openjdk-tests/compare/adoptium%3Aaqa-tests%3Amaster...mytest), [1 commit behind](https://github.com/llxia/openjdk-tests/compare/mytest...adoptium%3Aaqa-tests%3Amaster) adoptium/aqa-tests:master.

This file is already in the master branch, not needed in this PR.

**Comment by llxia** [comment]
The PR is rebased.

File: functional/OpenJcePlusTests/build.xml (hunk 1 above)
Comment (JasonFengJ9) [reply functional/OpenJcePlusTests/build.xml:31]: Apparently it's my workspace ignoring this file, sorry for the confusion.

**Review by JasonFengJ9 — APPROVED** [review]

**Pull request merged**